docker-compose.yml

```diff
@@ -6,3 +6,6 @@ services:
             dockerfile: Dockerfile
         env_file:
             - .env
+
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
```

Comment:
What does this do? Shouldnt this also contain consul?

Reply:
host.docker.internal would link to the host, I wasn't too sure how to link consul from the host into the container

Reply:
Why not make a consul container inside the compose file?